tests/typ/compiler/repr-color-gradient.typ

```diff
@@ -0,0 +1,22 @@
+// Test representation of values in the document.
+
+---
+// Colors
+#set page(width: 400pt)
+#set text(0.8em)
+#blue \
+#color.linear-rgb(blue) \
+#oklab(blue) \
+#cmyk(blue) \
+#color.hsl(blue) \
+#color.hsv(blue) \
+#luma(blue)
+
+---
+// Gradients
+#set page(width: 400pt)
+#set text(0.8em)
+#gradient.linear(blue, red) \
+#gradient.linear(blue, red, dir: 45deg) \
+#gradient.linear(blue, red, dir: 45deg, relative: "self") \
+#gradient.linear(blue, red, dir: 45deg, space: rgb)
```

Comment:
plz add newlines at the end of the test files :smiley:

Reply:
This is not something that is consistent across the code base, hence why I didn’t, if you want to go ahead but I have spent way more than enough time on this as it is to get bogged down in minute, nonsensical, nitpicks

Reply:
Sorry, I had just driven for two hours, I was in quite a mood.

And I thought of this: `cargo fmt` strips the last line, shouldn't it be the same for tests?

Reply:
I don't `cargo fmt` does that? At least not in my setup. It is a bit inconsistent across the code base because I don't bring it up anymore in PRs.